### PR TITLE
Restructuring tools

### PR DESCRIPTION
**Pull request by nsivabalan**
Some tools were using classes from Store package and hence they had to be made public just for the sake of tools. But making it public, will cause anyone use it which is uncalled for. So, this patch restructures those tools into store package within tools project and hence they can be made non-public.

Reviewers: Gopal, Casey
SLA: 10 mins

./gradlew build && ./gradlew test passed

### CODE REVIEW

**Comment by vgkholla** [comment]
@cgtz please pull into an IDE and check the structures. If you want context/intent please talk to me or @nsivabalan

File: ambry-tools/src/main/java/com.github.ambry.store/HardDeleteVerifier.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
  * distributed under the License is distributed on an "AS IS" BASIS,
```
Comment (cgtz) [inline ambry-tools/src/main/java/com.github.ambry.store/HardDeleteVerifier.java]: This class is package-private but has a main method. I guess this is valid, and the main method is accessible, but it seems a bit odd.

Comment (nsivabalan) [inline ambry-tools/src/main/java/com.github.ambry.store/HardDeleteVerifier.java]: Can't change it to public as the class name is different from file name. @pnarayanan : Do you remember the reason why is it so. Or can I go ahead and change the class name

**Comment by cgtz** [comment]
@nsivabalan one other thing: it seems like in ambry-api, packages with a common part of the path are all in one folder (so, there is one com.github.ambry folder with other folders inside of it). Here, there are two separate folders for com.github.ambry.store and com.github.ambry.tools.

File: ambry-tools/src/main/java/com.github.ambry.store/HardDeleteVerifier.java (hunk 1 above)
Comment (cgtz) [inline ambry-tools/src/main/java/com.github.ambry.store/HardDeleteVerifier.java]: @nsivabalan It seems like you already changed the file name to match the class name, right?

**Comment by nsivabalan** [comment]
@cgtz : Good point. Have fixed it.

**Pull request merged**